orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/model/HealthState.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.model;
+
+// adapted from clouddriver's com.netflix.spinnaker.clouddriver.model.HealthState
+public enum HealthState {
+  Failed,
+  Down,
+  OutOfService,
+  Unknown,
+  Starting,
+  Succeeded,
+  Up;
+
+  // HealthState.valueOf does:
```

Comment:
make this into a javadoc `/**` ?

Reply:
does it matter? In what way is a javadoc better?

Reply:
I feel like in this case you're describing what this method does/returns and Javadoc is exactly for that (and will show up in IDE)